portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,7 +36,57 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Chris!</h1>");
+    // Prepare query.	 
+    Query query = new Query("Comment").addSort("timestamp", SortDirection.DESCENDING);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    int maxComments = Integer.parseInt(request.getParameter("maxComments"));		
+    
+    // Get desired number of comments.		
+    List<String> comments = new ArrayList<>();
+    int currComments = 0;
```

Comment:
(super) nit: Consider renaming this to something like `numComments`. We want our variable names to describe the values they contain. IMO `currComments` implies a list of comments, rather than a number.

Reply:
fixed